### DISCUSSION TITLE
Dashboard workflow list

### DISCUSSION THREAD

**Discussion by mesfandi** [elsa-workflows/elsa-core#1082] [Q&A]
Hi,
I am wondering if there is any way that I can send filtered data to the dashboard and only list certain workflows in the dashboard when it loads (not all the workflows exist in the database). I want to list the workflows related to a certain user. 
thanks.

**Comment by sfmskywalker** [comment]
Not out of the box. Elsa doesn't have a concept of users; this is something your application can provide.

The question then becomes: "how to associate a user with a workflow?"

One way could be to handle the "WorkflowSaved" event and associate the currently signed-in user with the workflow, either by storing the user ID in the workflow's `Tag` property or even `CustomAttributes` dictionary, or in your own database table called e.g. "UserWorkflows" where you map a workflow definition ID to a user ID.

Your application can then query all workflow definition IDs for a given user ID, and then user these workflow definition IDs to bulk-query all workflow definitions.
